src/languageservice/services/yamlCompletion.ts

```diff
@@ -122,6 +122,11 @@ export class YAMLCompletion extends JSONCompletion {
     const collector: CompletionsCollector = {
       add: (suggestion: CompletionItem) => {
         let label = suggestion.label;
+        if (!label) {
+          // we receive not valid CompletionItem as `label` is mandatory field, so just ignore it and report to telemetry
+          this.telemetry.sendTrack('yaml.completion.missing.label', { data: suggestion });
```

Comment:
How is this telemetry event useful? Remember sending events has an actual cost. If this is not actionable, then why even send it?